documentation/asciidoc/computers/config_txt/autoboot.adoc

```diff
@@ -0,0 +1,40 @@
+== `autoboot.txt`
+
+`autoboot.txt` is an optional configuration file that can be used to specify the
+`boot_partition` number. This is sometimes used with NOOBS to bypass the boot menu
+selection and boot a specific partition.
+In the latest firmware this can also be used in conjunction with the `tryboot` feature to implement A/B booting for OS upgrades.
```

Comment:
Should "the latest firmware" be changed to "firmware newer than XXXX" ?

Reply:
I'll remove 'latest firmware' ... with the bootloader docs we found this just bit-rotted. I think the place for that sort of stuff is on the PIP e.g. For A/B booting you will need Bullseye release X or newer.

Reply:
Updated to remove 'latest firmware'

We are now just waiting for the rpi-update 'master' firmware release. Once that's done I'll ping this forum thread to direct early adopters to the bleeding edge software with added docs :)
https://forums.raspberrypi.com/viewtopic.php?p=2046776&hilit=tryboot#p2046776